collections/georef/batchgeoreftool.php

```diff
@@ -383,7 +399,8 @@
 								</fieldset>
 							</div>
 							<div style="margin:15px;">
-								<table>
+								<table class='xdebug-error xe-deprecated' dir='ltr' style='border: 1px solid black;'>
```

Comment:
This whole table needs to be replaced with grid-like elements per [our discussion on slack the other day](https://symbiota-support.slack.com/archives/D06E4HD96JF/p1708730334701399?thread_ts=1708627799.608139&cid=D06E4HD96JF).

Reply:
Also, `xdebug-error` classes shouldn't be added in. Their presence (presumably in the Total Validator Pro [TVP from now on] pdf I sent you?) just means that the debugger found an issue and rendered a new table in the DOM. Kinda tricky. In this case, this table made by the debugger probably came from deprecation warnings with the `trim()` method that I fixed in [this commit](https://github.com/BioKIC/Symbiota/pull/962/commits/5a2892cfd6d259ef050d658394c9f9ccfa7cf4ee).